src/main/kotlin/com/example/pass_manager/service/ClientServiceImpl.kt

```
@@ -0,0 +1,65 @@
+package com.example.pass_manager.service
+
+import com.example.pass_manager.domain.MongoClient
+import com.example.pass_manager.exception.ClientAlreadyExistsException
+import com.example.pass_manager.exception.ClientNotFoundException
+import com.example.pass_manager.repositories.ClientRepository
+import com.example.pass_manager.repositories.PassRepository
+import org.bson.types.ObjectId
+import org.springframework.data.repository.findByIdOrNull
+import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.time.Instant
+
+@Service
+class ClientServiceImpl(
+    private val clientRepository: ClientRepository,
+    private val passRepository: PassRepository
+) : ClientService {
+    override fun findById(clientId: ObjectId): MongoClient? = clientRepository.findByIdOrNull(clientId)
```

Maybe it's better to throw an exception here?
We should return nulls only at repository level